contrib/util/k3s-killall.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
```

Comment:
We should perhaps change this so that we can use the packaged binaries from the path, instead of the host /bin/sh
```suggestion
#!/usr/bin/env sh
```